src/lib.rs

```diff
@@ -312,7 +375,9 @@ mod tests {
     #[test]
     fn test_pomodoro_check_and_switch() {
         // Given
-        let mut pomodoro = Pomodoro::new((0, 2), (0, 2));
+
+        let sound = default_sound_path();
+        let mut pomodoro = Pomodoro::new((0, 3), (0, 2) ,sound, true);
```

Comment:
Important: here as well, the test fails because the times were updated, but the sleeping times weren't, causing the timer to be in an unexpected state when we assert it. It can be fixed using: 
```suggestion
        let mut pomodoro = Pomodoro::new((0, 2), (0, 2) ,sound, true);
```

Reply:
This works 👍.